opal/mca/btl/usnic/Makefile.am

```diff
@@ -103,7 +105,8 @@ libmca_btl_usnic_la_LIBADD = $(opal_ofi_LIBS)
 
 if OPAL_BTL_USNIC_BUILD_UNIT_TESTS
 usnic_btl_run_tests_CPPFLAGS = \
-    -DBTL_USNIC_RUN_TESTS_SYMBOL=\"opal_btl_usnic_run_tests\"
+    -DBTL_USNIC_RUN_TESTS_SYMBOL=\"opal_btl_usnic_run_tests\" \
+    -DOMPI_LIBMPI_NAME=\"$(OMPI_LIBMPI_NAME)\"
```

Comment:
Should this really be `$(AM_CPPFLAGS)`?

Reply:
great question.

Reply:
I think it should.  I'll add another commit here.